spark/src/main/scala/org/apache/spark/sql/delta/TypeWidening.scala

```diff
@@ -62,6 +62,19 @@ object TypeWidening {
       case _ => false
     }
 
+  /**
+   * Returns whether the given type change is eligible for **automatic** widening. Only a subset of
```

Comment:
What is automatic widening? Is it a well established term?

Reply:
I reworded to "type change during schema evolution"